pom.xml

```diff
@@ -251,7 +255,7 @@
     <version.org.reflections>0.9.11</version.org.reflections>
     <!-- Avoid using scannotation in favor of its successor org.reflections -->
     <version.org.scannotation>1.0.3</version.org.scannotation>
-    <version.org.slf4j>1.7.26</version.org.slf4j>
+    <version.org.slf4j>1.7.30</version.org.slf4j>
```

Comment:
I doubt there is any trouble with this, but not sure if we should check with @mbiarnes or someone else to be sure this is ok.